src/collections/_documentation/workflow/notifications/workflow.md

```diff
@@ -3,12 +3,99 @@ title: 'Workflow'
--   Regressions – when Sentry changes state of an issue from **resolved** to **unresolved**
+By default, these notifications are sent to anyone who is a member of the project the issue occurs in. Still, each member may choose their participation state, as well as opt-in or opt-out of notifications for specific issues. See [Unsubscribe](#unsubscribe) below for more details.
 
-By default, these notifications are sent to anyone who is a member of the project the issue occurs in, but each member may choose their participation state, as well as opt in or opt out of notifications for specific issues (in the "Notifications" section at the bottom right of the issue detail view). 
+## Issue States
 
-Read about [notification management]({%- link _documentation/workflow/notifications/notification-management.md -%}) for more information.
+Sentry provides a few states for each issue, which significantly impacts how notifications work.
+
+### Unresolved
+
+- The default state when an issue is added to the system.
+
+### Resolved
+
+- An issue is marked as resolved when an individual resolves it by manually changing the state on the issue page UI, submitting a fix, or when the project’s auto-resolve feature is configured. Team members are notified via email. Auto-resolve is located in **Project Settings > General Settings**.
+
+[{% asset notifications/auto-resolve.png alt="Event settings sliding toggle for disabling auto resolve." %}]({% asset notifications/auto-resolve.png @path %})
+
+- Resolved means the issue has stopped firing events. If another event occurs, the issue reverts to unresolved.
+
+### Regressions
+
+- When Sentry changes the state of an issue from resolved to unresolved. If a resolved issue begins experiencing more events, it will revert to unresolved. Regressions trigger a workflow alert.
+- Keep in mind; regression emails look very similar to other Notification emails.
+
+## Core Notifications
+
+### Assigned
+
+- A team member is assigned the issue, and they're notified via email, even if the issue is resolved.
+
+- Assignment auto subscribes you to workflow notifications.
+
+[{% asset notifications/workflow_notifications_subscription.png alt="Workflow notifications subscription buttons. Options include always, only on issues I subscribe to, and never." %}]({% asset notifications/workflow_notifications_subscription.png @path %})
+
+### Unassigned
+
+- A team member has not been assigned the issue.
+
+### Comments
+
+- A team member adds a new comment to an issue's Comments thread.
+
+## Digests
+
+Sentry will automatically digest alerts sent by some services to avoid flooding your inbox with individual issue notifications. To control how frequently notifications are delivered, use the sliders.
+
+- Minimum delivery interval
+- Maximum delivery interval
+
+[{% asset notifications/digest_intervals.png alt="Intervals that manipulate timing of digests." %}]({% asset notifications/digest_intervals.png @path %})
+
+## Notification Management
```

Comment:
damn now that it's removed I feel like it should be its own section 😅. Things like project notifications don't make sense on the "workflow" page. Is the content too small to elicit its own page do you think? Maybe this could just be on the main page?

Reply:
Maybe on the main page? I dunno. Certain notification management details only apply to alerts or workflow. Where would you recommend putting the project notifications on the main page and leave anything for alerts or workflow on their own separate pages?